CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ minor release, the component will be purged, so be prepared (see `Updating` sect
 - `neofs-cli object nodes` command to get SNs for an object (#2512)
 - Fetching container estimations via iterators to prevent NeoVM stack overflow (#2173)
 - `neofs-adm morph netmap-candidates` CLI command (#1889)
+- `neofs-lens object get` CLI command (#1336)
```

Comment:
@roman-khimov, @cthulhu-rider, what do you think about such a commands row:
- `neofs-lens meta ...`
- `neofs-lens blobovnicza ...`
- `neofs-lens peapod ...`
- `neofs-lens write-cache ...`

... and the current `neofs-lens object ...`. mb add some subcommand?

Reply:
we have `inspect` command in all component sections. I'd call supercomponent as `storage` and
```suggestion
- `neofs-lens storage inspect` CLI command (#1336)
```